### PR TITLE
Add event commands

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds docs for the following commands

events:create-event-provider
events:subscribe
events:unsubscribe
events:list

## Affected pages

-  ...

## Links to related PRs or Jira tickets

-  ...

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My changes follow the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/events/commands.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+The `events:list` command returns a list of subscribed events.
+
+### Usage
+
+`bin/magento events:list`
+
+### Example
+
+```bash
+bin/magento events:create-event-provider
```
Comment (jhadobe) [inline src/pages/events/commands.md:108]:
```suggestion
bin/magento events:list
```

File: src/pages/events/commands.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+### Example
+
+```bash
+bin/magento events:unsubscribe observer.catalog_product_save_after
+```
+
+### Response
+
+```terminal
+Successfully unsubscribed from the `observer.catalog_product_save_after` event"
```
Comment (jhadobe) [inline src/pages/events/commands.md:94]:
```suggestion
Successfully unsubscribed from the `observer.catalog_product_save_after` event
```

**Review by jhadobe — CHANGES_REQUESTED** [review]

File: src/pages/events/commands.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+description: Learn about the commands needed to set up eventing on an Adobe Commerce instance.
+---
+
+# Eventing commands
+
+Adobe I/O Events allow you to receive notifications of real-time events taking place in Adobe services.
+
+## Create an event provider
+
+The `events:create-event-provider` command generates an event provider ID. Add the generated ID as the value of the **Stores** > Configuration > **Adobe Services** > **Adobe IO Events** > **Adobe I/O Event Provider ID** field in the Commerce Admin.
```
[the remark this answers is not in the document]
Comment (keharper) [reply src/pages/events/commands.md:12]: This is correct. "Configuration" is not a clickable part of the path, so by convention, it's not bolded.

File: src/pages/events/commands.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+The `events:subscribe` command subscribes the current provider to the specified event. You must define the event code using the following pattern:
+
+```text
+<type>.<event_name>
+```
+
+where:
+
+-  `type` specifies the origin of the event. This value must be either `observer` or `plugin`.
+-  `event_name` identifies the event to subscribe. For example: `catalog_product_save_after`.
```
[the remark this answers is not in the document]
Comment (keharper) [reply src/pages/events/commands.md:48]: It doesn't exist yet.

**Review by jhadobe — APPROVED** [review]

File: src/pages/events/commands.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (oshmyheliuk) [reply src/pages/events/commands.md:12]: `command generates an event provider ID` - I think it's better to say that `command creates an event provider in Adobe IO Events and returns it ID`.

**Review by oshmyheliuk — APPROVED** [review]

**Pull request merged**